### PR TITLE
support system prune

### PR DESCRIPTION
**Pull request by junnplus**
Signed-off-by: Ye Sijun <junnplus@gmail.com>

### CODE REVIEW

File: cmd/nerdctl/system_prune.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	if err := networkPrune(cmd, client, ctx); err != nil {
+		return err
+	}
+	if vFlag {
+		if err := volumePrune(cmd, client, ctx); err != nil {
+			return err
+		}
+	}
+	return imagePrune(cmd, client, ctx)
+}
```
Comment (AkihiroSuda) [inline cmd/nerdctl/system_prune.go:103]: Can we prune builder cache too?
Can be another PR though

Comment (junnplus) [reply cmd/nerdctl/system_prune.go:103]: Yep, I will make another PR for it.

**Review by AkihiroSuda — APPROVED** [review]
Thanks
